DEFINITION.md

```diff
@@ -124,3 +124,13 @@ usaha yang minimal.
 **Cloud Native Computing Foundation** berusaha untuk mendorong adopsi paradigma ini dengan membina dan menopang sebuah
 ekosistem proyek-proyek yang bersifat sumber terbuka (*open source*) dan tidak memihak vendor tertentu. Kami mendemokratisasi pola-pola terkini agar
 inovasi-inovasi ini dapat diakses oleh semua orang.
+
+
+## Türkçe
+
+
+Bulut mevzun teknolojiler, kuruluşların ölçeklenebilir uygulamaları genel, özel ve hibrit bulutlar gibi modern, dinamik ortamlarda oluşturup çalıştırmasını sağlar. Konteynerlar; hizmet ağları, mikro hizmetler, sabit altyapı ve bildirime dayalı API'ler bu yaklaşımı temsil etmektedir.
```

Comment:
dinamik -> değişken
konteynerlar -> konteynerler
mikro hizmetler -> mikro servisler

ek olarak "bulut mevzun" çok anlaşılabilir gelmedi ama yerini alabilecek bir önerim yok maalesef

Reply:
konteynerler olduguna emin misin? benim gordugum orneklerde hep lar ile bitiyordu. "ler" ile pek okunakli durmuyor gibi :)

Reply:
Evet, buna küçük ünlü uyumu deniyor, Türkçe'nin temellerinde mevcut bir kural seti. Konteyner yabancı bir kelime olduğu için uyum aramayabilirsin ancak yine de okurken ve söylerken dil aşinalığı sağlamak açısından kelimeye eklenecek ekin ses uyumlu halini kullanmakta fayda var.

-ler yerine -lar ekleyebilmek için ekten önceki harf kalın ünlü olmalı (a/ı/o/u), bu sebeple konteynır yazmadığın durumda -lar eğreti duruyor.

Konteyner şeklinde yazında küçük ünlü uyumuna uyuyor, bu durumda da son sesli harf e olduğu için çoğul ekinin -ler olması gerekiyor.

Reply:
"bulut mevzun"un pek anlasilir olmasina katiliyorum. Her ne kadar diger ceviriler de "Cloud Native" diye kullanilmis olsa da "Bulut kokenli (Cloud Native) teknolojiler" olarak kullanilmasi daha anlasilir olabilir. "Bulut yerlisi" da dogru olabilir.

Reply:
Sanirim en uygunu "Bulut tabanli" demek olacak

Reply:
"bulut yerlisi" şirinmiş :)

1:1 tercümesi olmadığını düşünerek aşağıdakilerden hangisi daha yakın anlamlı geliyor size?

cloud based -> bulut tabanlı
cloud native -> bulut yerlisi
cloud compatible -> bulut uyumlu - buluta uyumlu

Reply:
'Bulut yerlisi' egrelti duruyor:), bence bulut tabanli iyi oldu. update edip, commit yaptim